manager.go

```diff
@@ -909,6 +910,27 @@ func (m *Manager) DetachHook(id ProbeIdentificationPair) error {
 	return nil
 }
 
+// DetachAndDeleteHook - Detaches an eBPF program from a hook point and delete it from the manager.
+func (m *Manager) DetachAndDeleteHook(id ProbeIdentificationPair) error {
+	idToDelete := -1
+	var err error
+	for mID, mProbe := range m.Probes {
+		if mProbe.Matches(id) {
+			if err = mProbe.Stop(); err != nil {
+				return fmt.Errorf("couldn't stop probe %s: %w", id, err)
+			}
```

Comment:
Is this missing a `mProbe.Detach` call, like the `Detach` function?

Reply:
Calling `mProbe.Stop()` does the same thing as `mProbe.Detach()` + it closes the underlying eBPF program. So in this specific case it should be enough.